app/assets/javascripts/app/views/poll_view.js

```diff
@@ -21,11 +21,21 @@ app.views.Poll = app.views.Base.extend({
     var originalPostLink = isReshare && this.model.get('root') ?
       '<a href="/posts/' + this.model.get('root').id + '" class="root_post_link">' + Diaspora.I18n.t('poll.original_post') + '</a>' :
       '';
+    var answerGiven = this.model.get("poll_participation_answer_id");
+    var currentUser = defaultPresenter.current_user;
+
+    if (defaultPresenter.poll && defaultPresenter.poll.poll_answers) {
+      var pollAnswers = defaultPresenter.poll.poll_answers;
+      pollAnswers = _.map(pollAnswers, function(answer) {
+        return _.extend(answer, {isCurrentUserVote: answer.id === answerGiven, currentUser: currentUser});
+      });
```

Comment:
The only purpose of these four lines is changing `defaultPresenter.poll.poll_answers` and I'd like to make that more explicit. Maybe something like this:

```JavaScript
defaultPresenter.poll.poll_answers.forEach(function (answer) {
  // set answer.isCurrentUserVote and answer.currentUser
});
```